aemcmc/__init__.py

```diff
@@ -2,6 +2,12 @@
 
 __version__ = _version.get_versions()["version"]
 
+
+from aemcmc.basic import construct_sampler
+
+# isort: off
 # Register rewrite databases
 import aemcmc.conjugates
 import aemcmc.gibbs
+
+# isort: on
```

Comment:
Why have this before the other imports (or in any particular order)?

Reply:
I naively copied [what `AePPL` does](https://github.com/aesara-devs/aeppl/blob/main/aeppl/__init__.py), i.e. importing the files that register rewrites last. Is there something wrong with this?

I don't see a particular reason to import `aemcmc.conjugates` before `aemcmc.gibbs` besides the conventional order of imports.

Reply:
It's nice to have the expected order of operations stated explicitly in code, but Python should successfully handle the import ordering itself&mdash;in most cases, at least.  It could become a problem as the library grows and the initialization order and/or cross-module dependencies get more complicated, though.  This is what we had to deal with during the early Theano refactoring.